bin/putasset.js

```diff
@@ -79,7 +79,8 @@ function main() {
         owner,
         tag,
         filename,
-    }).catch(log);
+    }).catch(log).then(dl_url=> console.log(dl_url));
+
```

Comment:
What point of additional logging?

Reply:
It is neat to provide an url where the newly uploaded asset can be downloaded